sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorLiveTestBase.cs

```diff
@@ -82,58 +82,66 @@ protected async Task<DisposableDataFeed> CreateTempDataFeedAsync(MetricsAdvisorA
             return await DisposableDataFeed.CreateDataFeedAsync(adminClient, dataFeed);
         }
 
+        protected int Count(DimensionKey dimensionKey)
```

Comment:
We have a convenient `Count` method to make sure that we are getting the right amount of dimension columns back from the service. It doesn't feel that useful to users, so I haven't added it directly to `DimensionKey`. It's an additive change, so we can always add it if we get user feedback.